projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -26,12 +27,28 @@ class REST_Controller {
 	public static $namespace = 'jetpack/v4/stats-app';
 
 	/**
+	 * Hold an instance of WPCOM_Stats.
+	 *
+	 * @var WPCOM_Stats
+	 */
+	protected $wpcom_stats;
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->wpcom_stats = new WPCOM_Stats();
+	}
+
+	/**
+
 	 * Registers the REST routes for Stats.
 	 *
 	 * @access public
 	 * @static
 	 */
 	public function register_rest_routes() {
+		// Stats for single resource type.
 		register_rest_route(
 			static::$namespace,
 			sprintf( '/sites/%d/stats/(?P<resource>[\-\w]+)/(?P<resource_id>[\d]+)', Jetpack_Options::get_option( 'id' ) ),
```

Comment:
I think we should get rid of the `sites/$d` part. This makes no sense outside of WPCOM.